src/components/studies/scheduler/Scheduler.tsx

```diff
@@ -809,6 +813,8 @@ const Scheduler: React.FunctionComponent<SchedulerProps> = ({
         </DialogTitle>
         <DialogContent style={{padding: 0}}>
           <ConfigureBurstTab
+            hasBursts={hasBursts}
+            setHasBursts={setHasBursts}
```

Comment:
I would rename setHasBursts to onSetHasBursts to make clear that it's a callback Fn. Otherwise -- looks good